generators/firechip/src/main/scala/TargetConfigs.scala

```diff
@@ -85,8 +85,46 @@ class WithFireSimDesignTweaks extends Config(
 
 // Tweaks to modify target clock frequencies / crossings to firesim defaults
 class WithFireSimDefaultFrequencyTweaks extends Config(
```

Comment:
I feel like these frequency selection configs aren't FireSim specific and could probably go into the chipyard package.

Reply:
I think the testchip one is a reasonable one to go in Chipyard. The FireSim based one (3.2GHz) makes sense to stay here since it determines the 200GB/s NIC. The default one can probably also go in Chipyard... but since it is a default it should stay here no?